README.md

```diff
@@ -1,179 +1,109 @@
-# TinyGo - Go compiler for microcontrollers
+# TinyGo - Go compiler for small places
 
-[![Build Status](https://travis-ci.com/aykevl/tinygo.svg?branch=master)](https://travis-ci.com/tinygo-org/tinygo)
+[![Build Status](https://travis-ci.com/tinygo-org/tinygo.svg?branch=dev)](https://travis-ci.com/tinygo-org/tinygo)
 
-> We never expected Go to be an embedded language and so it's got serious
-> problems [...].
+TinyGo is a Go compiler intended for use in small places such as microcontrollers, WebAssembly (WASM), and command-line tools.
 
--- Rob Pike, [GopherCon 2014 Opening Keynote](https://www.youtube.com/watch?v=VoS7DsT1rdM&feature=youtu.be&t=2799)
-
-TinyGo is a project to bring Go to microcontrollers and small systems with a
-single processor core. It is similar to [emgo](https://github.com/ziutek/emgo)
-but a major difference is that I want to keep the Go memory model (which implies
-garbage collection of some sort). Another difference is that TinyGo uses LLVM
-internally instead of emitting C, which hopefully leads to smaller and more
-efficient code and certainly leads to more flexibility.
+It uses the [Go compiler toolchain](https://golang.org/pkg/go/) alongside [LLVM](http://llvm.org) to provide an alternative way to compile programs written in the Go programming language.
 
-My original reasoning was: if [Python](https://micropython.org/) can run on
-microcontrollers, then certainly [Go](https://golang.org/) should be able to and
-run on even lower level micros.
-
-Example program (blinky):
+Here is an example program that blinks the built-in LED when run directly on an Arduino Uno microcontroller:
```

Comment:
This code is actually supported on many different boards, basically all boards that have a built-in LED. What about this?

> Here is an example program that blinks the built-in LED when run directly on any supported board with onboard LED: